ui/src/album/AlbumGridView.js

```diff
@@ -53,7 +54,15 @@ const useStyles = makeStyles((theme) => ({
   },
   albumSubtitle: {
     fontSize: '12px',
-    color: theme.palette.type === 'dark' ? '#c5c5c5' : '#696969',
+    color: themeColorSelector(
```

Comment:
I don't get what this is doing... Not straightforward. Take a look at the comment below, maybe just retrieveing this colour from the theme is a better choice.